_data/vulnerability_scanning_tools.yml

```diff
@@ -38,8 +38,8 @@
   owner: Arachni	
   license: Free for most use cases	
   platforms: Most platforms supported
-- title: AVDS
-  url: https://www.scanmyserver.com/
+- title: beSECURE (Formally: AVDS)

```

Comment:
```suggestion
- title: beSECURE (formerly: AVDS)

```

No worries. I've seen that typo a number of times lately. Apple or Samsung must have added a rule to their auto-correct or predictive text or something.....